x/multi-staking/types/key.go

```diff
@@ -31,3 +36,28 @@ var (
 	// mem store key
 	CompletedDelegationsPrefix = []byte{0x00}
```

Comment:
it looks like this key if just for the mem store, and the ones above are for the persisted store. should we still use a unique prefix? this one is the same as `BondTokenWeightKey`

Reply:
I'll make it different just to be sure

Reply:
@catShaark will you be doing that in this pr?